flycheck.el

```diff
@@ -3072,7 +3084,7 @@ nil."
                      "")))
                 (`interrupted "-")
                 (`suspicious "?"))))
-    (concat " FlyC" text)))
+    (concat " " flycheck-mode-line-lighter-base text)))
```

Comment:
Fine by me :) I'm just used to seeing either single-character lighters with no spaces, and multi-character lighters with a space.